app/src/main/resources/application.properties

```diff
@@ -34,6 +35,8 @@ quarkus.index-dependency.jaxrs.artifact-id=jboss-jaxrs-api_2.1_spec
 %dev.registry.streams.topology.num.standby.replicas=1
 %dev.registry.streams.topology.processing.guarantee=exactly_once
 %dev.registry.streams.topology.replication.factor=1
+%dev.registry.streams.topology.global.id.topic=global-id-topic
+%dev.registry.streams.topology.storage.topic=storage-topic
```

Comment:
Shouldn't this be something like this:

```
 %dev.registry.streams.topology.storage.topic=${REGISTRY_STREAMS_TOPOLOGY_STORAGE_TOPIC:storage-topic}
```

Reply:
Looks like env vars are picking up without to be explicitly defined as described in #637 by @alesj 
I tried with latest image build on top of master and https://github.com/Apicurio/apicurio-registry-demo and it was working fine with custom topic names
I can test it again, but looks like this should working fine

Reply:
OK maybe it's something Quarkus is doing by parsing the ENV variables.  I see the name of the env variable mirrors the property name.  Or maybe I'm missing something else.  But if it works and you've tested it, that's fine for now.  👍